doc/bc/changes-7.4.md

```diff
@@ -0,0 +1,11 @@
+# Backwards compatibility changes
+
+Changes affecting version compatibility with former or future versions.
+
+## Changes
+
+* New content/translate Policy has been added. You can use new `eZ/Bundle/EzPublishMigrationBundle/Command/LegacyStorage/AddContentTranslatePolicyCommand` to add new Policy to existing Roles. See documentation for more details.
```

Comment:
Maybe change this to the actual command people can run, but you'll also need to mention how they enable the bundle then...\*


\* We should probably move away from having a hidden bundle for this, at least add commented out link in Kernel.php for it, or even enabled by default in dev env.